doc/rbd/iscsi-initiator-rhel.rst

```diff
@@ -0,0 +1,89 @@
+------------------------------------------------
+The iSCSI Initiator for Red Hat Enterprise Linux
+------------------------------------------------
+
+**Prerequisite:**
+
+-  Package ``iscsi-initiator-utils-6.2.0.873-35`` or newer must be
+   installed
+
+-  Package ``device-mapper-multipath-0.4.9-99`` or newer must be
+   installed
+
+**Installing:**
+
+Install the iSCSI initiator and multipath tools:
+
+   ::
+
+       # yum install iscsi-initiator-utils
+       # yum install device-mapper-multipath
+
+**Configuring:**
+
+#. Create the default ``/etc/multipath.conf`` file and enable the
+   ``multiapthd`` service:
+
+   ::
+
+       # mpathconf --enable --with_multipathd y
+
+#. Add the following to ``/etc/multipath.conf`` file:
+
+   ::
+
+       devices {
+               device {
+                       vendor                 "LIO-ORG"
+                       hardware_handler       "1 alua"
+                       path_grouping_policy   "failover"
+                       path_selector          "queue-length 0"
+                       failback               60
+                       path_checker           tur
+                       prio                   alua
+                       prio_args              exclusive_pref_bit
+                       no_path_retry          120
```

Comment:
I think to be safe we should just change this to 

no_path_retry queue

It will then emulate the default rbd/librbd behavior.